File: CS/WindowsFormsApplication86/MyTokenEditViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.ViewInfo;

namespace WindowsFormsApplication86 {
    public class MyTokenEditViewInfo : TokenEditViewInfo {
        public MyTokenEditViewInfo(MyRepositoryItemTokenEdit properties)
            : base(properties) {
        }
        protected override TokenEditTokenEvaluator CreateTokenEvaluator() {
            return new MyTokenEditTokenEvaluator(this);
        }
    }
}

File: VB/WindowsFormsApplication86/MyTokenEditViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.ViewInfo

Namespace WindowsFormsApplication86
	Public Class MyTokenEditViewInfo
		Inherits TokenEditViewInfo
		Public Sub New(ByVal properties As MyRepositoryItemTokenEdit)
			MyBase.New(properties)
		End Sub
		Protected Overrides Function CreateTokenEvaluator() As TokenEditTokenEvaluator
			Return New MyTokenEditTokenEvaluator(Me)
		End Function
	End Class
End Namespace
